traitlets/traitlets.py

```diff
@@ -352,6 +352,8 @@ class that does the following:
     """
 
     name = None
+    bequeaths = ()
```

Comment:
What does `bequeaths` accomplish that `inherits` alone does not? Do we really need to specify an attribute as inheritable? Can we try a version with just `inherits`?